filesystem/local/filesystem_test.go

```
@@ -0,0 +1,200 @@
+	"context"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var ctx = context.Background()
+
+func TestFilesystem_basic(t *testing.T) {
+	// init
+	os.Mkdir("./testfile/basic", os.ModePerm)
```

golangci-lint (1.64.8)

Error return value of `os.Mkdir` is not checked

(errcheck)